tests/vanflow_snooper.py

```diff
@@ -0,0 +1,405 @@
+#!/usr/bin/env python
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import argparse
+import json
+import logging
+import sys
+from threading import Thread
+
+from proton.handlers import MessagingHandler
+from proton.reactor import Container
+from proton import Message
+
+
+# These mappings come directly from the vanflow.h source. They will need to be
+# updated as new records/attributes/etc are added
+#
+
+record_types = {
+    0: "SITE",
+    1: "ROUTER",
+    2: "LINK",
+    3: "CONTROLLER",
+    4: "LISTENER",
+    5: "CONNECTOR",
+    6: "FLOW",
+    7: "PROCESS",
+    8: "IMAGE",
+    9: "INGRESS",
+    10: "EGRESS",
+    11: "COLLECTOR",
+    12: "PROCESS_GROUP",
+    13: "HOST",
+    14: "LOG",
+}
```

Comment:
Consider using an Enum for this (`from enum import Enum`). It is nice idiomatic python that I think will work here.

Reply:
Nice!